rest/route/github.go

```diff
@@ -199,6 +203,42 @@ func (gh *githubHookApi) Run(ctx context.Context) gimlet.Responder {
 		owner := *event.Repo.Owner.Login
 		repo := *event.Repo.Name
 		PRNum := *event.Issue.Number
+		user := *event.Comment.User.Login
+
+		isAuthorizedArgs := data.UserRepoPair{
+			Username: user,
+			Owner:    owner,
+			Repo:     repo,
+		}
```

Comment:
I'd combine lines 203- with this declaration and just reference the data in the args to avoid all the extra strings.

Reply:
done.